README.md

```diff
@@ -1,39 +1,72 @@
-# LINQ to DB LINQPad Driver [![build status](https://ci.appveyor.com/api/projects/status/github/linq2db/linq2db.LINQPad)](https://ci.appveyor.com/project/igor-tkachev/linq2db-linqpad)
+TODO: add azure/nuget badges
 
-linq2db.LINQPad is a driver for [LINQPad](http://www.linqpad.net) that supports the following databases:
+# LINQ to DB LINQPad Driver
 
-- **DB2** (LUW, z/OS) *(client software must be installed)*
-- **DB2 iSeries** (using [3rd-party provider](https://github.com/LinqToDB4iSeries/Linq2DB4iSeries)) *(iAccess 7.1+ software must be installed)*
+linq2db.LINQPad is a driver for [LINQPad](http://www.linqpad.net). Supported versions: LINQPad 5 and LINQPad 6.
+
+Following databases supported (by both LINQPad 5 and LINQPad 6 if not noted):
+
+- **DB2** (LUW, z/OS)
+- **DB2 iSeries** (using [3rd-party provider](https://github.com/LinqToDB4iSeries/Linq2DB4iSeries)) *(iAccess 7.1+ software must be installed)*. **IMPORTANT:** currently available only for LINQPad 5 using lin2db.LINQPad version 2.9.3 or earlier
 - **Firebird**
-- **Informix** *(client software must be installed)*
-- **Microsoft Access**
-- **Microsoft Sql Azure**
-- **Microsoft Sql Server** 2000+
-- **Microsoft SqlCe**
-- **MySql**
-- **Oracle** *(client software must be installed if native driver is used)*
+- **Informix**
+- **Microsoft Access** *(supports both OleDb and ODBC)*
+- **Microsoft Sql Server** 2000+ *(including **Microsoft Sql Azure**)*

```

Comment:
The current set of tests cannot access SQL Server 2000 on .Net Core due to an explicit prohibition by the underlying provider.  If there's two different supported Sql Server providers, I'm not sure which one it tests.  As LINQPad 6 runs on .Net Core, it may not actually support SQL Server 2000.

Reply:
Interesting.. Does it apply to v2000 only?

Reply:
Ok, found it here https://stackoverflow.com/questions/51779884/asp-net-core-2-0-connect-to-sql-server-2000-error-message-the-sql-server-instan

Reply:
I think we can remove sql2k support from linq2db/tests for .net core then